src/plugins/items/capes/skillcape-emotes.ts

```diff
@@ -0,0 +1,34 @@
+import { lockEmote, unlockEmote } from '@server/plugins/buttons/player-emotes-plugin';
+import { ActionType, RunePlugin } from '@server/plugins/plugin';
+import { equipAction } from '@server/world/actor/player/action/equip-action';
+
+export const skillcapes: Array<number> = [
```

Comment:
Also not really required but could you move these to itemids?

Reply:
I'll add them when I get off work later today!

Reply:
Finally found the time to fix the issue